### ISSUE TITLE
[Bug]: VanillaVistas, Repurposed Structures, and Biome mod. Eye of ender crash

### ISSUE THREAD

**Issue by Enterprise12nx01** [TelepathicGrunt/RepurposedStructures#318]
I reported the issue on OTG's github, here is the link: https://github.com/PG85/OpenTerrainGenerator/issues/1004
i talked to them on discord a bit as well

**Comment by TelepathicGrunt** [comment]
I do not support 1.16.5 sorry. Likely what they are doing is they change the structure locate code in a way that doesn’t do proper checks first to make sure the given incoming structure can actually spawn in the dimension before doing the search. I don’t really remember 1.16.5 code anymore but basically, a check is needed to make sure the biomes that the structure can actually spawn in is possible to actually generate in the world. 

Basically, they made this chunk generator method likely searching forever for a nether stronghold instead of checking if dimension can actually spawn the nether biomes that holds the structure first
https://github.com/TelepathicGrunt/RepurposedStructures/blob/99e8d656dbada6b85bed262e3276c092859c89be/src/main/java/com/telepathicgrunt/repurposedstructures/utils/StructureLocator.java#L20

**Comment by TelepathicGrunt** [comment]
To verify it is an issue on otg’s end, try a new world with the same mods as before but without otg. If the eyes of ender works, then otg changed a behavior somewhere with the chunk generator’s structure locate code

**Comment by Enterprise12nx01** [comment]
> To verify it is an issue on otg’s end, try a new world with the same mods as before but without otg. If the eyes of ender works, then otg changed a behavior somewhere with the chunk generator’s structure locate code

i did test with otg and it works fine. its the otg vanilla presets that specifically has an issue!
thank you for responding even though you dont support the version!

**Comment by Enterprise12nx01** [comment]
Further testing, it only breaks with RS, OTG vanilla vistas and BOP(BYG works fine, who knows if other biome mods have issues)
